### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by daroczig** [MangoTheCat/goodpractice#29]
When I try to install the pkg as described in the README, I get this error:

```r
> source("https://install-github.me/MangoTheCat/goodpractice")
Downloading GitHub repo MangoTheCat/goodpractice@master

Error: Malformed remote specification 'MangoTheCat/cyclocomp'
```

Although it's probably a https://install-github.me issue -- but I suppose it's managed by you :) Pls feel free to close this issue, of course the package can be installed after installing the dependencies.

**Comment by gaborcsardi** [comment]
While I have seen this before, I cannot reproduce it now. Any hints? E.g. `devtools::session_info`?

**Comment by daroczig** [comment]
This is on my personal computer:

```r
> source("https://install-github.me/MangoTheCat/goodpractice")
Downloading GitHub repo MangoTheCat/goodpractice@master
Error: Malformed remote specification 'MangoTheCat/cyclocomp'
> devtools::session_info()
Session info -------------------------------------------------------------------
 setting  value                       
 version  R version 3.3.1 (2016-06-21)
 system   x86_64, linux-gnu           
 ui       X11                         
 language (EN)                        
 collate  en_US.UTF-8                 
 tz       Europe/Budapest             
 date     2016-09-19                  

Packages -----------------------------------------------------------------------
 package  * version date       source        
 devtools * 1.12.0  2016-06-24 CRAN (R 3.3.1)
 digest     0.6.9   2016-01-08 CRAN (R 3.3.1)
 memoise    1.0.0   2016-01-29 CRAN (R 3.3.1)
 withr      1.0.2   2016-06-20 CRAN (R 3.3.1)
```

But I remember I first faced this problem on my work computer, will check there as well.

**Comment by gaborcsardi** [comment]
OK, I can indeed reproduce it on Linux, thanks!

**Comment by gaborcsardi** [comment]
Thanks again! Just fixed this here: 4f721343c3b0c63aa2da9ea29313ac0236883e1f

**Comment by daroczig** [comment]
Thanks a lot, installs without any problems now.
